packages/logger/src/logBuffer.ts

```diff
@@ -110,8 +110,12 @@ export class CircularMap<V> extends Map<string, SizedSet<V>> {
 
     const buffer = this.get(key) || new SizedSet<V>();
 
-    if (buffer.currentBytesSize + item.byteSize >= this.#maxBytesSize) {
+    if (
+      buffer.currentBytesSize &&
```

Comment:
Can we be explicit here and check exactly rather than relying on type coercion: `!== 0`.